spec/Observable-spec.ts

```diff
@@ -946,3 +946,57 @@ describe('Observable.lift', () => {
     }
   });
 });
+
+if (Symbol && Symbol.asyncIterator) {
+  describe('async iterator support', () => {
+    it('should work for sync observables', async () => {
+      const source = of(1, 2, 3);
+      const results: number[] = [];
+      for await (const value of source) {
+        results.push(value);
+      }
+      expect(results).to.deep.equal([1, 2, 3]);
```

Comment:
By definition of usage of the `await` keyword, this runs on the next microtask tick, right? Is that worth highlighting?